react-client/dist/style.css

```diff
@@ -200,15 +209,17 @@
   max-width: 1500px;
   display: grid;
   grid-template-columns: auto 400px ;
-  padding-bottom: 0px;
-  margin-bottom: 0px;
+  padding-bottom: 5px;
+  margin-bottom: 20px;
   font-family: Arial, Helvetica, sans-serif;
+  margin-top: 0px;
 }
 
 .grid-container-helpful{
+  margin-top: 999px;
   max-width: 200px;
   display: grid;
-  grid-template-columns: 65px 75px 75px 65px;
+  grid-template-columns: auto auto auto auto;
```

Comment:
interesting change. I would love to the the page.

Reply:
this is so that the display changes with the size of the screen instead of static